README.md

```diff
@@ -11,16 +13,57 @@ about writing tests.
 
 ## Running tests
```

Comment:
Might be good to rename this to something, more clear like running load tests. And add a small development section that includes how to run the unit tests.